tools/ci_build/github/azure-pipelines/linux-ort-pytorch-ci-pipeline.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     submodules: recursive
 
   - script: >
-      python -m ensurepip
+      sudo apt-get install -y python3-pip
```

Comment:
Most pipelines seems to be creating docker images to run build / tests. By installing packages directly on the server, we can break the whole environment. Each host can be ubuntu 16.04, ubuntu 18.04, centos, etc, so package version/availability could differ, which can cause hard to repro issues.

Is it possible to use `tools/ci_build/github/linux/run_dockerbuild.sh` to create a docker image and `docker run` to execute the tests as other pipelines? In these images you can install everything you need without compromising the host server

https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-e2e-test-ci-pipeline.yml

and

https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend-test-ci-pipeline.yml

might serve as examples

Reply:
Thanks for this suggestion, this is very helpful. I'm not familiar with setting up docker pipelines. I'm doing some trial and error editing the template and finding out how it works.

Reply:
@thiagocrepaldi 
I've added the step to run docker build, but when running the test script I'm seeing this error:
ModuleNotFoundError: No module named 'onnxruntime'

Do you know if another step is needed to install the wheel?

Reply:
Hi @neginraoof I am not sur ewhether we spoke about this, but you are probably missing a pip install <ort>/build/Linux/<build_type>/dist/onnxruntime*.whl to install ort on your docker image

another possibility is using `export PYTHONPATH:<ort>/build/Linux/<build_type>:$PYTHONPATH` before running your script